01.multithreading/MultiThreading.Task6.Continuation/Program.cs

```diff
@@ -7,24 +7,66 @@
    Demonstrate the work of the each case with console utility.
 */
 using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace MultiThreading.Task6.Continuation
 {
     class Program
     {
+        const int TaskAmount = 10;
         static void Main(string[] args)
         {
-            Console.WriteLine("Create a Task and attach continuations to it according to the following criteria:");
-            Console.WriteLine("a.    Continuation task should be executed regardless of the result of the parent task.");
-            Console.WriteLine("b.    Continuation task should be executed when the parent task finished without success.");
-            Console.WriteLine("c.    Continuation task should be executed when the parent task would be finished with fail and parent task thread should be reused for continuation.");
-            Console.WriteLine("d.    Continuation task should be executed outside of the thread pool when the parent task would be cancelled.");
-            Console.WriteLine("Demonstrate the work of the each case with console utility.");
-            Console.WriteLine();
+            // Console.WriteLine("Create a Task and attach continuations to it according to the following criteria:");
+            // Console.WriteLine("a.    Continuation task should be executed regardless of the result of the parent task.");
+            // Console.WriteLine("b.    Continuation task should be executed when the parent task finished without success.");
+            // Console.WriteLine("c.    Continuation task should be executed when the parent task would be finished with fail and parent task thread should be reused for continuation.");
+            // Console.WriteLine("d.    Continuation task should be executed outside of the thread pool when the parent task would be cancelled.");
+            // Console.WriteLine("Demonstrate the work of the each case with console utility.");
+            // Console.WriteLine();

```

Comment:
This task is about continuation options, all criteria can be completed separately, please read the following https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskcontinuationoptions?view=net-8.0&source=recommendations

something like this:
`// Continuation task should be executed regardless of the result of the parent task
            var parentTask1 = Task.Run(() => DoWork());
            parentTask1.ContinueWith(t => Console.WriteLine("Continuation 1"), TaskContinuationOptions.None);
            // Continuation task should be executed when the parent task finished without success
            var parentTask2 = Task.Run(() => throw new Exception());
            parentTask2.ContinueWith(t => Console.WriteLine("Continuation 2"), TaskContinuationOptions.NotOnRanToCompletion); etc.....`